### PR TITLE
Update framework

### PR DESCRIPTION
**Pull request by peppy**

### CODE REVIEW

File: osu.Game/Screens/Edit/Timing/ControlPointTable.cs [hunk 1]
```diff
@@ -104,10 +104,11 @@ protected override void LoadComplete()
                     // child items valid coordinates from the start, so ballpark something similar
                     // using estimated row height.
                     var row = Items.FlowingChildren.SingleOrDefault(item => item.Row.Equals(val.NewValue));
+
                     if (row == null)
                         return;
 
-                    float minPos = Items.GetLayoutPosition(row) * row_height;
+                    float minPos = row.Y;
```
Comment (smoogipoo) [inline osu.Game/Screens/Edit/Timing/ControlPointTable.cs:111]: Is this correct? @bdach or @EVAST9919 can answer better than me. The suggestion in the PR was `Items.IndexOf(row) * row_height` but this _reads_ like it should be okay?

Comment (peppy) [reply osu.Game/Screens/Edit/Timing/ControlPointTable.cs:111]: it works correctly is all i know

Comment (bdach) [reply osu.Game/Screens/Edit/Timing/ControlPointTable.cs:111]: I think it'll work incorrectly for rows that aren't loaded in, but will double check.

Comment (bdach) [reply osu.Game/Screens/Edit/Timing/ControlPointTable.cs:111]: Seems fine in testing actually and I can explain to myself as to why (the flow that no longer is there was the main problem here), so...

**Review by bdach — APPROVED** [review]

**Pull request merged**